server/views/pages/adjustments/additional-days/remove-prospective.njk

```diff
@@ -6,11 +6,8 @@
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block content %}
+  <a href="/{{ prisoner.prisonerNumber }}" class="govuk-back-link">Back</a>
```

Comment:
The URL for the backlink was fine. The "Cancel" button is the one that should go to the adjustments hub.